src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -292,6 +292,9 @@ public bool CapturePointer(Pointer value)
 			else
 			{
 				_pointCaptures.Add(value);
+#if  __IOS__
```

Comment:
should this apply to Android as well?

Reply:
@MakanWG wasn't sure. We only observed regressions on iOS and lookign at the code it is only used 
[thumb.cs](src\Uno.UI\UI\Xaml\Controls\Primitives\Thumb.cs) for .net and MacOS.  We could apply it everywhere except wasm to be safe.

Reply:
Checked with @carldebilly , ended up just mirroring the behavior of wasm on iOS to avoid inconsistencies and regressions.
There is definitely work to do to harmonize the behavior for all platforms though.